content/docs/iac/concepts/vs/cloud-template-transpilers/aws-cdk/_index.md

```diff
@@ -34,6 +34,41 @@ Choosing the right infrastructure as code tool is important, and we want you to
+
+### Speed
+
+Pulumi enables faster development and deployment workflows compared to CDK due to its direct integration with cloud providers without going through a transpiler.
+
+- __Deployment__. Pulumi communicates directly with cloud providers, bypassing the intermediate step of generating CloudFormation templates as required by CDK. Since CDK relies on CloudFormation as its deployment engine, it inherits many of CloudFormation's speed limitations. (see [Pulumi vs. CloudFormation](/docs/concepts/vs/cloud-templates/cloudformation/)).
+- __Development__. Pulumi's runtime engine allows for immediate feedback during deployment, enabling developers to debug and iterate more quickly. In contrast, CDK's reliance on CloudFormation introduces delays due to template transpilation and longer error resolution times.
+- __Testing__. Pulumi supports fast, in-memory unit tests that mock external calls, enabling developers to validate infrastructure without deploying it. CDK testing is limited to assertions against synthesized CloudFormation templates, which slows down the feedback loop.
+
+### Security
+
+Pulumi natively integrates security features with infrastructure provisioning.
+
+- __Secrets Management__. Pulumi has built-in secrets management that encrypts secrets both in transit and at rest. It also integrates seamlessly with external secrets managers like AWS Secrets Manager. CDK has no built-in support for managing or handling encrypted secrets.
+- __Policy as Code__. Pulumi enables your organization to enforce security and compliance policies during deployment using Pulumi CrossGuard's policy as code. This feature is tightly integrated into Pulumi's workflows, whereas CDK offers only limited support for such policies. __[NEED TO VALIDATE]__
+
+### Savings
+
+Pulumi can deliver cost savings over CDK due to its multi-cloud support and efficiency gains in the inner development loop.
+
+- __Multi-Cloud Support__. Pulumi supports over 160 cloud providers, allowing your organization to optimize costs by choosing the most cost-effective provider for your needs. CDK is limited to AWS, which may lock you into higher cost AWS services. Using CDK will also necessitate employing another IaC system to manage non-AWS resources which will increase training time for new engineers and operational complexity.
+- __Operational Efficiency__. Pulumi’s faster deployments reduce engineering time spent on debugging and iteration cycles. This translates into lower development time costs compared to the more time-intensive development workflows required by CDK.
+- __Interoperability__. Pulumi can integrate with existing resources managed by other IaC tools, eliminating the need for costly migrations or extensive code refactoring. CDK lacks interoperability with other IaC tools, which requires full migration to CDK code in order to interact with existing resources.
+
+### Scale
```

Comment:
This should probably be the first category, since it is the largest source of real differentiation our audience cares about. Instead of "Provider Ecosystem" which sounds theoretical, I would be very upfront about multi-cloud and making sure the fact that multi-cloud also includes SaaS providers pops, since that's a major CDK weakness.